### PR TITLE
Update to Wasm Bootstrapper 1.0.5

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

Fixes mixed path file copy

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Directory.Build.targets [hunk 1]
```diff
@@ -17,8 +17,8 @@
     <PackageReference Update="Uno.SourceGeneration" Version="1.32.0-dev.297" />
     <PackageReference Update="Uno.Core" Version="1.29.0-dev.93" />
     <PackageReference Update="Uno.Core.Build" Version="1.29.0-dev.93" />
-    <PackageReference Update="Uno.Wasm.Bootstrap" Version="1.0.3" />
-    <DotNetCliToolReference Update="Uno.Wasm.Bootstrap.Cli" Version="1.0.3" />
+    <PackageReference Update="Uno.Wasm.Bootstrap" Version="1.0.4" />
```
Comment (carldebilly) [inline src/Directory.Build.targets]: `1.0.4` or `1.0.5`? You should adjust the title of the PR ;-)

Comment (jeromelaban) [reply src/Directory.Build.targets]: Really was 1.0.5, mixed up the version numbers :)

**Pull request merged**